jobs/JEVS_GLOBAL_ENS_PREP

```diff
@@ -38,16 +38,17 @@ export DCOMIN=${DCOMIN:-${DCOMROOT}}
 export COMINccpa=${COMINccpa:-$(compath.py $envir/com/ccpa/${ccpa_ver})}
 export COMINcmce=$COMINnaefs
 export COMINprepbufr=${COMINprepbufr:-$(compath.py $envir/com/obsproc/${obsproc_ver})}
-export COMINsnow=${DCOMIN}
```

Comment:
Also, COMINprepbufr needs to be changed to COMINobsproc.  This is because obsproc is the package in production, not prepbufr.  This needs to be changed.